sparkpost/transmissions.py

```diff
@@ -39,7 +39,13 @@ def _translate_keys(self, **kwargs):
             kwargs.get('use_draft_template', False)
         model['content']['reply_to'] = kwargs.get('reply_to')
         model['content']['subject'] = kwargs.get('subject')
-        model['content']['from'] = kwargs.get('from_email')
+        if kwargs.get('from_name'):
+            model['content']['from'] = {
+                'name': kwargs.get('from_name') or '',
```

Comment:
this isn't necessary because you've already checked it's truthy-ness in line 42. I commented that without noticing that one and I deleted after I found it's not necessary. Sorry for confusion. 

You don't need to create another PR. Just commit and push the same branch which will auto update the PR. Thanks again. 

Can you also possibly add unit tests covering you tests? `transmissions` module is now missing some coverage.